### ISSUE TITLE
Backup save game folder

### ISSUE THREAD

**Issue by stealthbell** [FezVrasta/ark-server-tools#1093]
Hi there,
 
I am wondering if I am missing something with the instance backup command as after inspecting the tar file created it does not seem to contain persistent mod info such as the information from Awesome Spyglass or DinoStorage2. I guess this would be required for a complete restore in the event of a server restoration attempt.
 
This data appears to be stored in the ShooterGame/Saved/SaveGames folder.
 
Perhaps I am just missing something like a flag for the backup command?
 
If not would it be possible to check for the presence of data in this folder and add it to the backup?
 
Cheers

**Comment by klightspeed** [comment]
What filename pattern do these files follow, so we can add them to the backup command?

**Comment by stealthbell** [comment]
Hi, thanks for the lightning quick response! Filenames seem to be *.sav but there is sub-folder structure also. I have popped a few images on imgur for you to have a look at which may be more informative than my description.

https://imgur.com/a/1G7xBxD

Many thanks

**Comment by klightspeed** [comment]
v1.6.52 includes `SavedGames` (as opposed to `SavedArks`) in the backup if it exists.

**Comment by stealthbell** [comment]
Hiya,
 
Pulled the latest update (1.6.52) and ran the backup command for the instance. Checking the backup with tar -tf doesn't seem to have the files still. To be sure I had updated I confirmed version with arkmanager --version and it states release tag 1.6.52.
 
Would I need to restart the instance in question for this update to take effect? Perhaps I am doing something else silly.
 
The command I am using to backup is arkmanager backup @Ragnarok
 
Many thanks.
